pypxe/tftp.py

```diff
@@ -18,18 +18,31 @@ class TFTPD:
     '''
     def __init__(self, **serverSettings):
         self.ip = serverSettings.get('ip', '0.0.0.0')
-        self.port = serverSettings.get('port', 69)
+        self.port = serverSettings.get('port', 79)
         self.netbootDirectory = serverSettings.get('netbootDirectory', '.')
-        self.mode_debug = serverSettings.get('mode_debug', False) #debug mode
+        self.logger = serverSettings.get('logger', None)
+        self.mode_debug = serverSettings.get('mode_debug', False)
         self.sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         self.sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
         self.sock.bind((self.ip, self.port))
 
-        if self.mode_debug:
-            print 'NOTICE: TFTP server started in debug mode. TFTP server is using the following:'
-            print '\tTFTP Server IP: {}'.format(self.ip)
-            print '\tTFTP Server Port: {}'.format(self.port)
-            print '\tTFTP Network Boot Directory: {}'.format(self.netbootDirectory)
+        if self.logger == None:
+            import logging
+            import logging.handlers
+            # setup logger
+            self.logger = logging.getLogger("tftp")
+            handler = logging.StreamHandler()
+            formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
+            handler.setFormatter(formatter)
+            self.logger.addHandler(handler)
+
+            if self.mode_debug:
+                self.logger.setLevel(logging.DEBUG)
+
+        self.logger.debug('TFTP server started in debug mode. TFTP server is using the following:')
+        self.logger.debug('\tTFTP Server IP: {}'.format(self.ip))
+        self.logger.debug('\tTFTP Server Port: {}'.format(self.port))
+        self.logger.debug('\tTFTP Network Boot Directory: {}'.format(self.netbootDirectory))
```

Comment:
In L43-L45 should we remove the `\t` characters since we're using the logger now? If using the logger perhaps we shouldn't use the `\t`? Thoughts?